### DISCUSSION TITLE
IPv6 Link Local addresses

### DISCUSSION THREAD

**Discussion by wilhelmy** [strongswan/strongswan#2067] [Q&A]
I read in the ancient archives that [StrongSWAN did not support IPv6 Link Local addresses](https://users.strongswan.narkive.com/G4YhjDAa/strongswan-strongswan-doesn-t-listen-to-ipv6-address#post2) 15 years ago. Is this still the case?

I want to authenticate network traffic with IPSec AH on my IPv6LL-only network.
Is this currently possible?

**Comment by tobiasbrunner** [comment]
Sounds like a duplicate of #1862.
